Makefile

```diff
@@ -17,64 +17,60 @@
 
 SHELL := /bin/bash
 # We assume an active virtualenv for development
-PYENV_REGEX := .pyenv/shims
-PY_BIN := python3
-# https://github.com/pypa/pip/issues/5599
-PIP_WRAPPER := $(PY_BIN) -m pip
-export PY39 := $(shell jq -r '.python_versions.PY39' .ci/variables.json)
-export PY310 := $(shell jq -r '.python_versions.PY310' .ci/variables.json)
-export PY311 := $(shell jq -r '.python_versions.PY311' .ci/variables.json)
-export PY312 := $(shell jq -r '.python_versions.PY312' .ci/variables.json)
-export HATCH_VERSION := $(shell jq -r '.prerequisite_versions.HATCH' .ci/variables.json)
-export HATCHLING_VERSION := $(shell jq -r '.prerequisite_versions.HATCHLING' .ci/variables.json)
-export PIP_VERSION := $(shell jq -r '.prerequisite_versions.PIP' .ci/variables.json)
-export WHEEL_VERSION := $(shell jq -r '.prerequisite_versions.WHEEL' .ci/variables.json)
 VIRTUAL_ENV ?= .venv
 VENV_ACTIVATE_FILE := $(VIRTUAL_ENV)/bin/activate
-VENV_ACTIVATE := . $(VENV_ACTIVATE_FILE)
-VEPYTHON := $(VIRTUAL_ENV)/bin/$(PY_BIN)
-PYENV_ERROR := "\033[0;31mIMPORTANT\033[0m: Please install pyenv.\n"
-PYENV_PREREQ_HELP := "\033[0;31mIMPORTANT\033[0m: please type \033[0;31mpyenv init\033[0m, follow the instructions there and restart your terminal before proceeding any further.\n"
-VE_MISSING_HELP := "\033[0;31mIMPORTANT\033[0m: Couldn't find $(PWD)/$(VIRTUAL_ENV); have you executed make venv-create?\033[0m\n"
-
-prereq:
-	pyenv install --skip-existing $(PY39)
-	pyenv install --skip-existing $(PY310)
-	pyenv install --skip-existing $(PY311)
-	pyenv install --skip-existing $(PY312)
-	pyenv local $(PY312)
-	@# Ensure all Python versions are registered for this project
-	@ jq -r '.python_versions | [.[] | tostring] | join("\n")' .ci/variables.json > .python-version
-	-@ printf $(PYENV_PREREQ_HELP)
-
-venv-create:
-	@if [[ ! -x $$(command -v pyenv) ]]; then \
-		printf $(PYENV_ERROR); \
-		exit 1; \
-	fi;
-	@if [[ ! -f $(VENV_ACTIVATE_FILE) ]]; then \
-		eval "$$(pyenv init -)" && eval "$$(pyenv init --path)" && $(PY_BIN) -mvenv $(VIRTUAL_ENV); \
-		printf "Created python3 venv under $(PWD)/$(VIRTUAL_ENV).\n"; \
-	fi;
+
+PY_VERSION = 3.12.2
```

Comment:
I would still use `.ci/variables.json` for this, as docs and IT keep reading this file. Although `prerequisite_versions` section from `.ci/variables.json` could go away I think.